server/util/clickhouse/schema/schema.go

```diff
@@ -226,6 +226,10 @@ type Execution struct {
 
 	Experiments []string `gorm:"type:Array(LowCardinality(String))"`
 
+	// If an executionID is retried, the RunID will differ for each retry attempt.
+	// Can be unset for executions that aren't retried.
+	RunID string
```

Comment:
(Forgot to mention this in the last round, sorry)

- Can we use ClickHouse's `uuid` type here? I think it will use a more compact storage format while still allowing queries/inserts to work in the same way.
- Should we also have an `is_retry` field for easier querying?